README.md

```diff
@@ -1,3 +1,24 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+Welcome to the NC news repository! This is the backend project for northcoders which will be used for displaying news?. Follow the instructions below to set up your local environment and get started.
+
+## Prerequisites
+
+Before you begin, make sure you have the following installed:
+
+- Node.js 
+
+## Setting Up Environment Variables
+
+To run this project locally, you need to set up the following environment variables:
+
+1. `PGDATABASE`: nc_news.
+2. `PGDATABASE`: nc_news_test.
+
+Follow these steps to set up the environment variables:
+
+1. Clone this repository to your local machine.
+2. Navigate to the project directory: `cd BE-NC-NEWS`.
+3. Create a new file named `.env` in the root directory.
+4. Open the `.env` file in a text editor of your choice.
+5. Add your environment variables 
```

Comment:
There are some spelling/syntax issues here that need to be resolved here - 
You need to have 2 separate .env files
`PGDATABASE`: nc_news.  should be 'PGDATABASE=nc_news' in .env.development
`PGDATABASE`: nc_news_test. should be 'PGDATABASE=nc_news_test' in .env.test

`cd BE-NC-NEWS` does not exist - your repo is called 'nv-news'